src/controllers/controller.cpp

```diff
@@ -96,6 +97,16 @@ void Controller::send(QList<int> data, unsigned int length) {
 }
 
 void Controller::receive(const QByteArray data, mixxx::Duration timestamp) {
```

Comment:
Hm, I don't think so. A lot of controllers send constant timing info (e.g. MIDI clock signals).